src/cromshell/status/command.py

```diff
@@ -0,0 +1,124 @@
+import fileinput
+import json
+import logging
+import re
+import sys
+
+import click
+import jq
+import requests
+
+from cromshell.utilities import CromshellConfig, HTTPUtils, IOUtils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="status")
+@click.argument("workflow_id")
+@click.pass_obj
+def main(cromshell_config, workflow_id):
+    """Check the status of a Cromwell job UUID"""
+    if cromshell_config.is_verbose:
+        LOGGER.info("status")
+    else:
+        click.echo("status")
+
+    ret_val = 0
+
+    # Set cromwell server using submission file. Running the function below with passing only the workflow id
+    # overrides the default cromwell url set in the cromshell config file, command line argument, and
+    # environment. This takes place only if the workflow id is found in the submission file.
+    CromshellConfig.CromshellConfig.override_cromwell_config_server(
+        workflow_id=workflow_id
+    )
+
+    # Check if Cromwell Server Backend works
+    HTTPUtils.assert_can_communicate_with_server(cromshell_config.cromwell_server)
```

Comment:
I don't know if this is still necessary strictly since we should be able to process the results of the request directly better than we can in bash, but lets not do anything about it now.

Reply:
Leave it for now, maybe remove it later. add a time limit if needed.